src/Libraries/libSystem/libsystem_kernel/Libsyscall.xcconfig

```diff
@@ -0,0 +1,74 @@
+#include "<DEVELOPER_DIR>/Makefiles/CoreOS/Xcode/BSD.xcconfig"
```

Comment:
Could you please delete this file and the Xcode project as well? (You can use `git rm --cached` to keep the file on disk, but remove it from the PR.)